python/lsst/sconsUtils/state.py

```diff
@@ -180,6 +187,23 @@ def _initEnvironment():
 
     if env['debug']:
         env.Append(CCFLAGS = ['-g'])
+
+    #
+    # determine if EUPS is present
+    #
+
+    # --no-eups overides probing
+    # XXX is it possible to test python snippets as a scons action?
```

Comment:
Yeah, it looks like this is weirdly absent from SCons.  Here's a workaround from the GalSim team: https://github.com/GalSim-developers/GalSim/blob/releases/1.2/SConstruct#L915